application/src/main/java/org/togetherjava/tjbot/commands/MessageReceiver.java

```diff
@@ -18,16 +19,25 @@
  * is added to.
  */
 public interface MessageReceiver extends Feature {
+    /**
+     * The pattern used as a default value for {@link MessageReceiver#getChannelNamePattern()}.
+     */
+    Pattern ANY_PATTERN = Pattern.compile(".*");
+
     /**
      * Retrieves the pattern matching the names of channels of which this receiver is interested in
      * receiving sent messages from. Called by the core system once during the startup in order to
      * register the receiver accordingly.
      * <p>
      * Changes on the pattern returned by this method afterwards will not be picked up.
+     * <p>
+     * Defaults to .* (i.e. any channel).
      *
      * @return the pattern matching the names of relevant channels
      */
-    Pattern getChannelNamePattern();
+    default Pattern getChannelNamePattern() {
```

Comment:
i dont like this addition. how many benefit from this right now? only bytecodecommand?

the disadvantage is that future implementors will not directly see that they can implement this as well, since its defaulted and hence not suggested by intellij when implementing it. so beginners will likely misunderstand.

Reply:
removed the defaulting